rotkehlchen/data/importer.py

```diff
@@ -784,6 +784,74 @@ def import_blockfi_trades_csv(self, filepath: Path) -> Tuple[bool, str]:
                     return False, str(e)
         return True, ''
 
+    def _consume_binance_purchases(self, csv_row: Dict[str, Any]) -> None:
+        """
+        Consume the file containing only purchases by credit/debit card from Binance.
+        """
+        # TODO perhaps put the 'Completed' into a variable as it may change
+        if csv_row['Status'] != 'Completed':
+            log.debug(f'Ignoring not completed Binance purchase. {csv_row}')
+            return
+
+        timestamp = deserialize_timestamp_from_date(
+            date=csv_row['Date(UTC)'],
+            formatstr='%Y-%m-%d %H:%M:%S',
+            location='Binance',
+        )
+
+        amount, base_asset = csv_row['Final Amount'].split(' ')
```

Comment:
This here can raise a `ValueError` which will go unhandled and bubble up all the way to the top.

Can you handle it here and reraise as `DeserializationError`? Also bundle up the other places where this is needed in the same try/except. Something along the lines of:
```suggestion
        try:
            amount, base_asset = csv_row['Final Amount'].split(' ')
            quote_amount, quote_asset = csv_row['Amount'].split(' ')
            fee, fee_currency = csv_row['Fees'].split(' ')
        except ValueError as e:
            raise DeserializationError(f'Could not split amount/asset from entry {csv_row}') as e
```